commons/all/src/main/java/org/infinispan/commons/configuration/attributes/ConfigurationElement.java

```diff
@@ -87,6 +88,17 @@ public boolean matches(T other) {
       return true;
    }
 
+   public boolean matches(T other, Attribute<?> ... ignored) {
```

Comment:
All these ignored parts should be able to be removed if we move the match skip to the HashConfiguration.